### PR TITLE
Style redesign and credits

### PR DESCRIPTION
**Pull request by Naramsim**
In order to celebrate almost 2000 stars on our main Github repo, I made some changes to our pokeapi.co frontend to make it more appealing. I also modified some sections of the `/about` page, to give more credits to all the people who have worked/are working on the PokeAPI project.

Check out the changes here: https://staging.pokeapi.co

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function Footer({id}) {
         <footer id={id} className={styles.footer}>
             <div className={styles.container}>
                 <p>
-                    © 2013–2019{' '}
+                    © 2013–{new Date().getFullYear()}{' '}
```
[the remark this answers is not in the document]
Comment (Naramsim) [reply src/components/Footer/Footer.js:11]: We can just remove the copyright I guess

File: src/components/ApiExplorer/Input.module.scss [hunk 2]
```diff
@@ -39,15 +41,19 @@ $mobile-breakpoint: 30em;
...
     color: white;
     border: none;
+    border-top-right-radius: 10px;
+    border-bottom-right-radius: 10px;
     padding: 0.35em 0.75em;
     font-size: 1em;
     display: block;
+    width: 7em;
     min-width: max-content;
+    cursor: pointer;
```
[the remark this answers is not in the document]
Comment (Naramsim) [reply src/components/ApiExplorer/Input.module.scss:56]: I think it's a standard having the pointer on buttons, I don't know a single site that drops the pointers from their buttons.

File: src/global.scss [hunk 3]
```diff
@@ -72,3 +72,49 @@ html,
...
+
+::-webkit-scrollbar {
+    width: 8px;
+    background-color:   transparent;
+}
+
+::-webkit-scrollbar-thumb {
+    background-color:   #c3c3c3;
+    border-radius:      4px;
+}
```
[the remark this answers is not in the document]
Comment (Naramsim) [reply src/global.scss:93]: Ah, I just saw that I used the webkit ones! Good catch. I can make them a bit thicker if you think it's too difficult to use them. But anyway nowadays no one uses scrollbars anymore, usually, you use your scroll wheel or your fingers. I think I'll add a custom scrollbar only for desktop devices and not for mobile devices.

[the remark this answers is not in the document]
Comment (cmmartti) [reply src/global.scss:93]: It's not true that nobody uses scrollbars. I personally use them all the time, as do many others. It's irritating that Mac OS 10 hides them by default, but at least it's possible to properly enable them. Windows 10 does the same in some cases, but not in the browser.

 It is my opinion that we should not modify the scrollbars in any way. As it is, they already fit in with the styling of the website, and customizing them would be change for the sake of change.

File: src/components/Footer/Footer.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmmartti) [reply src/components/Footer/Footer.js:11]: I agree. Maybe just say "Created by Paul Hallet and other PokèAPI contributors around the world".

File: src/components/ApiExplorer/Input.module.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmmartti) [reply src/components/ApiExplorer/Input.module.scss:56]: It's certainly not a standard. As explained in that article, the standard is the pointer cursor is for links, and the arrow cursor for everything else, but this has been diluted significantly in recent years when people override the defaults. I still think there's value in maintaining the distinction, but it's not a hill I'm going to die on.

**Comment by phalt** [comment]
Hey, I've had a made two weeks building some COVID-related services for my local government, haven't had time to look at this! Will have a poke around now.

**Comment by Naramsim** [comment]
I fixed almost all @cmmartti, suggestions. I wanted to change the theme of the JSONViewer but it's so difficult to find themes online and it's a mess creating one.

**Pull request merged**